lib/counter_culture/counter.rb

```diff
@@ -182,7 +182,11 @@ def foreign_key_value(obj, relation, was = false)
         value = value.send(relation.shift)
       end
 
-      return value.try(relation_primary_key(original_relation, source: obj, was: was).try(:to_sym))
+      if value.is_a?(ActiveRecord::Associations::CollectionProxy)
+        value.map { |v| v.try(relation_primary_key(original_relation, source: obj, was: was).try(:to_sym)) }
+      else
+        value.try(relation_primary_key(original_relation, source: obj, was: was).try(:to_sym))
```

Comment:
Can we pull this out into a helper method like
```
def relation_primary_key_from_model(model, original_relation, obj, was)
  model.try(relation_primary_key(original_relation, source: obj, was: was).try(:to_sym)
end
```
Not sure if that's worth it, but it makes it clearer that the same thing is happening in both cases, just mapping in one.